LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceRegistry.cs

```diff
@@ -201,11 +201,11 @@ private JoinDeviceLoader GetOrCreateJoinDeviceLoader(IoTHubDeviceInfo ioTHubDevi
         /// <summary>
         /// Searchs for devices that match the join request.
         /// </summary>
-        public async Task<LoRaDevice> GetDeviceForJoinRequestAsync(string devEUI, string devNonce)
+        public async Task<LoRaDevice> GetDeviceForJoinRequestAsync(string devEUI, DevNonce devNonce)
```

Comment:
Since there was a check for null or empty device nonce before, wouldn't the right translation here be a nullable `DevNonce`?

Reply:
Valid point, but based on the usage it seems that we didn't need the nullability in the first place. Do you think there's a bug somewhere?